pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -153,10 +153,22 @@ func (self *SuggestionsHelper) getRemoteBranchNames(separator string) []string {
 	})
 }
 
+func (self *SuggestionsHelper) getRemoteBranchNamesWithoutRemotePrefix() []string {
+	return slices.FlatMap(self.model.Remotes, func(remote *models.Remote) []string {
+		return slices.Map(remote.Branches, func(branch *models.RemoteBranch) string {
+			return branch.Name
```

Comment:
Is there an easy way to remove duplicates from the suggestions? (If we have more than one remotes, there are bound to be some and - since they have no prefix - that looks weird in the suggestions window...)

Reply:
@moha-gh You can wrap the whole thing in `lo.Uniq`!

Reply:
Not needed anymore with the new approach, but thanks anyway :)